test/integration/features/cmd-config.feature

```diff
@@ -0,0 +1,238 @@
+@command @basic
+Feature: Minishift config subcommands
+Commands `minishift config [sub-command]` are used for storing 
+user defined options which changes default behaviour of Minishift.
+
+  Scenario: Config is empty
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift config unset iso-url" succeeds
+      And executing "minishift config unset memory" succeeds
+      And executing "minishift config unset cpus" succeeds
+     Then executing "minishift config view" succeeds
+      And stdout should be empty
```

Comment:
do we have to unset the config options?

Reply:
@gbraad For minishift not, but it would be useful for CDK - with this it could be directly used in downstream. But I guess I mixed it up a little bit :/